collect_app/src/main/java/org/odk/collect/android/utilities/MediaUtils.java

```diff
@@ -82,362 +60,8 @@ public void openFile(Context context, File file) {
         }
     }
 
-    public String getMimeType(String url) {
+    private String getMimeType(String url) {
```

Comment:
Done.

Reply:
I'm rolling back this. I tested that way and it doesn't work in all cases but the old one does at leas for the cases I tested. Oh Android you are so funny! 
I added some fallbacks just in case like in getFileExtensionFromUri() above.